src/App.js

```diff
@@ -66,7 +66,6 @@ const mapStateToProps = (state) => {
 
 class Root extends React.Component {
   componentDidMount() {
-    hathorLib.WebSocketHandler.setup();
```

Comment:
Why don't we have to call it anymore?

Reply:
We don't need the websocket ready when we mount the app, only when we start the wallet.

We are already calling `setup` in the lib when starting the wallet.

Reply:
By the way, we have issue #39 describing this problem.

Reply:
Great. Thanks! We should add to the description of the issue that it solves #39.